### ISSUE TITLE
Qovery Container Registry runtime error

### ISSUE THREAD

**Issue by JanmanX** [Qovery/terraform-provider-qovery#162]
When applying the `qovery_container_registry`, I get a runtime error:
```
resource "qovery_container_registry" "dockerhub" {
  organization_id = var.qovery_organization_id
  name            = "Dockerhub"
  kind            = "DOCKER_HUB"
  url             = "https://docker.io"
}
```

```
qovery_container_registry.dockerhub: Creating...                                                                                                                                                                                                              
╷                                                                                                                                                                                                                                                             
│ Error: Plugin did not respond                                                                                                                                                                                                                               
│                                                                                                                                                                                                                                                             
│ The plugin encountered an error, and failed to respond to the plugin6.(*GRPCProvider).ApplyResourceChange call. The plugin logs may contain more details.                                                                                                   
╵                                                                                                                                                                                                                                                             
╷                                                                                                                                                                                                                                                             
│ Error: Plugin did not respond                                                                                                                                                                                                                               
│                                                                                                                                                                                                                                                             
│ The plugin encountered an error, and failed to respond to the plugin6.(*GRPCProvider).ApplyResourceChange call. The plugin logs may contain more details.                                                                                                   
╵                                                                                                                                                                                                                                                             
                                                                                                                                                                                                                                                              
Stack trace from the terraform-provider-qovery_v0.14.1 plugin:                                                                                                                                                                                                
                                                                                                                                                                                                                                                              
panic: runtime error: invalid memory address or nil pointer dereference                                                                                                                                                                                       
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xc94b0e]                                                                                                                                                                                        
                                                                                                                                                                                                                                                              
goroutine 55 [running]:                                                                                                                                                                                                                                       
github.com/qovery/terraform-provider-qovery/qovery.ContainerRegistry.toUpsertRequest({{0x1, 0x0, {0x0, 0x0}}, {0x0, 0x0, {0xc000720180, 0x24}}, {0x0, 0x0, ...}, ...})                                                                                        
        github.com/qovery/terraform-provider-qovery/qovery/resource_container_registry_model.go:45 +0xee                                                                                                                                                      
github.com/qovery/terraform-provider-qovery/qovery.containerRegistryResource.Create({{_, _}}, {_, _}, {{{{0x1054028, 0xc000621950}, {0xdb0580, 0xc000621380}}, {0xc0004dc4e0, 0x0, ...}}, ...}, ...)                                                          
        github.com/qovery/terraform-provider-qovery/qovery/resource_container_registry.go:154 +0x158                                                                                                                                                          
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).CreateResource(0xc0002a8160, {0x104fbe0, 0xc000620d20}, 0xc000807410, 0xc0008073b0)                                                                                               
        github.com/hashicorp/terraform-plugin-framework@v0.14.0/internal/fwserver/server_createresource.go:97 +0x8ce                                                                                                                                          
github.com/hashicorp/terraform-plugin-framework/internal/fwserver.(*Server).ApplyResourceChange(0x104fbe0?, {0x104fbe0, 0xc000620d20}, 0xc000356780, 0xc000807580)                                                                                            
        github.com/hashicorp/terraform-plugin-framework@v0.14.0/internal/fwserver/server_applyresourcechange.go:54 +0x4a8                                                                                                                                     
github.com/hashicorp/terraform-plugin-framework/internal/proto6server.(*Server).ApplyResourceChange(0xc0002a8160, {0x104fbe0?, 0xc000620bd0?}, 0xc000356550)                                                                                                  
        github.com/hashicorp/terraform-plugin-framework@v0.14.0/internal/proto6server/server_applyresourcechange.go:52 +0x27b                                                                                                                                 
github.com/hashicorp/terraform-plugin-go/tfprotov6/tf6server.(*server).ApplyResourceChange(0xc0006e61e0, {0x104fbe0?, 0xc0006203f0?}, 0xc0006f41c0)                                                                                                           
        github.com/hashicorp/terraform-plugin-go@v0.14.0/tfprotov6/tf6server/server.go:816 +0x574                                                                                                                                                             
github.com/hashicorp/terraform-plugin-go/tfprotov6/internal/tfplugin6._Provider_ApplyResourceChange_Handler({0xea6740?, 0xc0006e61e0}, {0x104fbe0, 0xc0006203f0}, 0xc0006f4000, 0x0)                                                                          
        github.com/hashicorp/terraform-plugin-go@v0.14.0/tfprotov6/internal/tfplugin6/tfplugin6_grpc.pb.go:385 +0x170                                                                                                                                         
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0000cc000, {0x1054318, 0xc000174b60}, 0xc0002e6480, 0xc0000b7e00, 0x168e300, 0x0)                                                                                                                         
        google.golang.org/grpc@v1.48.0/server.go:1295 +0xb2b                                                                                                                                                                                                  
google.golang.org/grpc.(*Server).handleStream(0xc0000cc000, {0x1054318, 0xc000174b60}, 0xc0002e6480, 0x0)                                                                                                                                                     
        google.golang.org/grpc@v1.48.0/server.go:1636 +0xa2f                                                                                                                                                                                                  
google.golang.org/grpc.(*Server).serveStreams.func1.2()                                                                                                                                                                                                       
        google.golang.org/grpc@v1.48.0/server.go:932 +0x98                                                                                                                                                                                                    
created by google.golang.org/grpc.(*Server).serveStreams.func1                                                                                                                                                                                                
        google.golang.org/grpc@v1.48.0/server.go:930 +0x28a                                                                                                                                                                                                   
                                                                                                                                                                                                                                                              
Error: The terraform-provider-qovery_v0.14.1 plugin crashed!                                                                                                                                                                                                  
                                                                                                                                                                                                                                                              
This is always indicative of a bug within the plugin. It would be immensely                                                                                                                                                                                   
helpful if you could report the crash with the plugin's maintainers so that it                                                                                                                                                                                
can be fixed. The output above should help diagnose the issue. 
```

**Comment by JanmanX** [comment]
This is a possible duplicate of https://github.com/Qovery/terraform-provider-qovery/issues/153

**Comment by mzottolaqovery** [comment]
Hello @JanmanX, I will take a look at those issues

**Comment by mzottolaqovery** [comment]
Issue fixed in the [v0.14.4](https://github.com/Qovery/terraform-provider-qovery/releases/tag/v0.14.4)
